doc/whats_new/v1.2.rst

```diff
@@ -213,6 +213,16 @@ Changelog
 - |Fix| Fixed error message of :class:`metrics.coverage_error` for 1D array input.
   :pr:`23548` by :user:`Hao Chun Chang <haochunchang>`.
 
+:mod:`sklearn.multioutput`
+..........................
+
+- |API| Added boolean `verbose` flag to classes:
```

Comment:
I'll say this is a feature:

```suggestion
- |Feature| Added boolean `verbose` flag to classes:
```

According to our legend:

https://github.com/scikit-learn/scikit-learn/blob/79c21c57aa622b76a0174e852c7806d6e8c11e43/doc/whats_new/changelog_legend.inc#L11-L12

"API" is used for API changes.